gunicorn/app/base.py

```diff
@@ -147,26 +147,27 @@ def load_config(self):
             for k, v in cfg.items():
                 self.cfg.set(k.lower(), v)
 
-        if args.config:
-            self.load_config_from_file(args.config)
+        env_args = parser.parse_args(self.cfg.get_cmd_args_from_env())
+
+        if args.config or env_args.config:
```

Comment:
Should one override the other rather than allowing both of these branches?

Reply:
I agree that `env_args.config` should override `args.config`. We should probably document this at http://docs.gunicorn.org/en/latest/settings.html#settings